tests/helpers/fund-accs/run_full.go

```diff
@@ -138,18 +138,25 @@ func RunFullNode(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 
 	runenv.RecordMessage("full -> %d has this %s balance", initCtx.GroupSeq, bal.String())
 
-	nid, _ := hex.DecodeString("0c204d39600fddd3")
-	data := []byte("f1f20ca8007e910a3bf8b2e61da0f26bca07ef78717a6ea54165f5")
-	for i := 0; i < 10; i++ {
-		tx, err := nd.StateServ.SubmitPayForData(ctx, nid, data, 70000)
+	var nid namespace.ID
+	if runenv.StringParam("namespace-id") == "1" {
+		nid = common.DefaultNameId
+	} else {
+		nid = common.GetRandomNamespace()
+	}
```

Comment:
DRY
```go
func nameSpaceID(runenv *runtime.RunEnv) namespace.ID {
	if runenv.StringParam("namespace-id") == "1" {
		return common.DefaultNameId
	}
	return common.GetRandomNamespace()

Reply:
looks like it is used in several places too

Reply:
Done